dist/profile/templates/confluence/vhost.conf

```
@@ -2849,12 +2849,18 @@ RewriteRule "^/display/JENKINS/ZMQ\+Event\+Publisher\+Plugin$" "https://plugins.
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Zoom\+Plugin$" "https://plugins.jenkins.io/zoom" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
-RewriteRule "^/display/JENKINS/IBM\+zOS\+Connector$" "https://plugins.jenkins.io/zos-connector" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/IBM\+zOS\+Connector\+Plugin$" "https://plugins.jenkins.io/zos-connector" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/violation\+comments\+to\+gitlab\+Plugin$" "https://plugins.jenkins.io/violation-comments-to-gitlab" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/claim\+Plugin$" "https://plugins.jenkins.io/claim/" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/appcenter\+Plugin$" "https://plugins.jenkins.io/appcenter/" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/xml-job-to-job-dsl\+Plugin$" "https://plugins.jenkins.io/xml-job-to-job-dsl/" [NC,L,QSA,R=301]
```

The page https://wiki.jenkins.io/display/JENKINS/xml-job-to-job-dsl+Plugin

The page https://wiki.jenkins.io/display/JENKINS/XML+Job+to+Job+DSL exists on the wiki and needs to be the source of this redirect.
```suggestion
RewriteRule "^/display/JENKINS/XML\+Job\+to\+Job\+DSL$" "https://plugins.jenkins.io/xml-job-to-job-dsl/" [NC,L,QSA,R=301]
```

---

Fixed by #1591